### PR TITLE
Adding test case support for Composite Patch

### PR DESCRIPTION
**Pull request by aayush3011**
# Description
1. Adding test case support to validate composite patch for the Patch APIs.
2. Composite Patch means patching a nested object.

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/PatchTest.java [hunk 1]
```diff
@@ -19,8 +19,7 @@
 import java.util.Objects;
 import java.util.UUID;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.fail;
+import static org.assertj.core.api.Assertions.*;
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/PatchTest.java:22]: We don't use `*` imports, please change this back to individual imports.

**Review by kushagraThapar — APPROVED** [review]
Looks good with minor suggestions, thanks @aayush3011  for updating this test case.

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/PatchTest.java (hunk 1 above)
Comment (aayush3011) [reply sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/PatchTest.java]: Fixed.

**Pull request merged**